plugins/woocommerce-blocks/assets/js/interactivity/directives.js

```diff
@@ -298,12 +298,14 @@ export default () => {
 				( { suffix } ) => suffix === 'default'
 			);
 
-			useEffect( () => {
-				// Prefetch the page if it is in the directive options.
-				if ( link?.prefetch ) {
-					// prefetch( href );
-				}
-			} );
+			// For some reason this useEffect crashes in Preact internals in some cases.
```

Comment:
This stops a consistent crash that I see in my env. as mentioned its a no-op anyway right now so harmless to comment out.